rescript/types/_format.py

```diff
@@ -144,7 +145,7 @@ def _validate_(self, level):
         self._validate(record_count_map[level])
 
 
-class AlignedRNAFASTAFormat(RNAFASTAFormat):
+class AlignedRNAFASTAFormat(AlignedDNAFASTAFormat):
```

Comment:
[drive-by comment]

huh... looks like AlignedDNAFASTAFormat can be used with RNA:

https://github.com/qiime2/q2-types/blob/af9e2be5743f98db5ed87c81c23ac13756b8186e/q2_types/feature_data/_format.py#L166-L171

The validator should maybe be changed to dissalow this... and that validation code should just be copied over here with valid RNA characters only, instead of inheriting that class (or, alternatively, just put this type in q2-types now).

Reply:
Yeah, this was clearly a quick hack to see if I can implement a new type. 

Anyway, this is the problem with many of the `DNA` Types. They can actually understand RNA and DNA data. But other operations focus specify `skbio.DNA`. I am surprised no one caught this much earlier. I was undecided on how to handle this. 

I should just go ahead and fix all the DNA types (Remove 'U') and add the RNA types (disallow 'T') to q2-types, as you suggest.

Reply:
Okay, I am going to add to `q2-types`. It is not as difficult as I thought. May take me a couple of days. I'll come back to this PR once I can submit a good `q2-types` PR.

Reply:
Just an update. @ebolyen and I had a brief discussion about how to update and "clean up" q2-types, etc... I'll hold off on this until the both of us can chat again in far greater detail within the next month or so...

Reply:
thanks for the update @mikerobeson ! @misialq has actually been busily refactoring q2-types to make a generic FASTA format, see here for details:
https://github.com/qiime2/q2-types/pull/254

once that goes in it will be cleaner to create an `FeatureDate[AlignedRNASequence]` type... it would be worth refactoring this PR in anticipation of those changes to q2-types.